### PR TITLE
feat: Setup boiler plate of the application

### PR DESCRIPTION
**Pull request by sergiolucaco**
Issues: Boilerplate of the project and commits better stuctured

### CODE REVIEW

**Comment by davidlampon** [comment]
It looks like you are commiting a folder inside the project for the main structure files. It looks like you have a folder inside a folder.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
-# gympad
-This is a side project intended to be developed using React as a learn by doing and to power up one of my daily routines! 
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (davidlampon) [inline README.md]: Please, replace the generic README with this code, it's more like it should be as you already had some text on it: https://github.com/sergiolucaco/gympad/pull/9/files#diff-04c6e90faac2675aa89e2176d2eec7d8

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "devDependencies": {
+    "@commitlint/cli": "^7.3.2",
+    "@commitlint/config-conventional": "^7.3.1",
+    "@commitlint/travis-cli": "^7.3.2",
```
Comment (davidlampon) [inline package.json]: I think we don't need this but let's remove it until we get to the cd/ci part

File: travis.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+language: node_js
```
Comment (davidlampon) [inline travis.yml]: Maybe delete this for now. We'll focus on CD/CI after so we don't put anything unnecessary in the commit

**Comment by davidlampon** [comment]
The commit message doesn't seem good. What do you think changing it for something like `feat: Setup boiler plate of the application`?

**Comment by sergiolucaco** [comment]
> The commit message doesn't seem good. What do you think changing it for something like `feat: Setup boiler plate of the application`?

I will add that commit message when I remove all the Travis thing and update the Readme ! Thanks !

**Comment by davidlampon** [comment]
Btw, you can edit the final commit name (which is taken from the first commit I think) directly on github. On the title of this PR to the right you have the edit button and you can change it as you wish to fit the naming convention as it will be the first important commit in the master git history. I can show you how to rebase the commit history but it is better to do it right from the start.

**Comment by sergiolucaco** [comment]
All the changes added !

**Comment by sergiolucaco** [comment]
I cannot merge de pull request since -- >The base branch requires all commits to be signed more info -> https://help.github.com/articles/about-commit-signature-verification/

**Comment by davidlampon** [comment]
It look to me that this is configurable for master using github interface: https://help.github.com/articles/about-required-commit-signing/

**Pull request merged**